src/Components/Facility/TreatmentSummary.tsx

```diff
@@ -49,50 +53,56 @@ export default function TreatmentSummary({
-            {consultationData?.facility_name ?? ""}
-          </h2>
-
-          <h2 className="text-center text-lg">
-            {t("treatment_summary__heading")}
-          </h2>
-
-          <div className="text-right font-bold">{formatDate(date)}</div>
-
-          <div className="mb-5 mt-2 border border-gray-800">
-            <BasicDetailsSection
-              patientData={patientData}
-              consultationData={consultationData}
-            />
-
-            <ComorbiditiesSection patientData={patientData} />
-
-            <DiagnosisSection consultationData={consultationData} />
-
-            <InvestigationsSection consultationId={consultationId} />
-
-            <TreatmentSection consultationData={consultationData} />
+        <PrintPreview title="Treatment Summary">
+          <div id="section-to-print" className="mx-5">
 
-            <PrescriptionsSection consultationId={consultationId} />
-
-            <InstructionsSection consultationData={consultationData} />
-          </div>
+          <div className="mb-3 flex items-center justify-between p-4 ">
+          <h3>{consultationData?.facility_name}</h3>
+          <img className="h-10 w-auto" src={careConfig.mainLogo?.dark} alt="care logo" />
         </div>
+            <h2 className="text-center text-lg">
+              {t("treatment_summary__heading")}
+            </h2>
+
+            <div className="text-right font-bold">{formatDate(date)}</div>
+
+            <div className="mb-5 mt-2 border border-gray-800">
+              <BasicDetailsSection
+                patientData={patientData}
+                consultationData={consultationData}
+              />
+
+              <ComorbiditiesSection patientData={patientData} />
+
+              <DiagnosisSection consultationData={consultationData} />
+
+              <InvestigationsSection consultationId={consultationId} />
+
+              <TreatmentSection consultationData={consultationData} />
+
+              <PrescriptionsSection consultationId={consultationId} />
+
+              <InstructionsSection consultationData={consultationData} />
+            </div>
+            <div className="pt-12">
+        <p className="font-medium text-secondary-800">
+          Sign of the Consulting Doctor
+        </p>
+        <PatientDetail name="Name of the Consulting Doctor">
+          {consultationData?.treating_physician_object &&
+            formatName(consultationData?.treating_physician_object)}
+        </PatientDetail>
+        <p className="pt-6 text-center text-xs font-medium text-secondary-700">
+          Generated on: {formatDateTime(new Date())}
+        </p>
+        <p className="pt-1 text-center text-xs font-medium text-secondary-700">
+          This is a computer generated prescription. It shall be issued to the
+          patient only after the concerned doctor has verified the content and
+          authorized the same by affixing signature.
+        </p>
+      </div>  
```

Comment:
These were not there in treatment summary before right? Why was this added?

Reply:
my bad